### ISSUE TITLE
Investigate code monitoring unsubscriptions

### ISSUE THREAD

**Issue by lguychard** [sourcegraph/sourcegraph#16621]
For the first version of code monitoring, we won't add "unsubscribe" links to notification emails, and will simply link back to the code monitor. Since the email recipient for private monitors is necessarily the owner of the monitor, they can take appropriate action on the monitor itself (disable action or delete monitor).

A suggested future improvement is to add a "Stop receiving notifications for this monitor" link to code monitoring emails.

This issue tracks the investigation work that should be conducted, along with engineering, prior to moving forward with implementation:
- Does "Stop receiving notifications" refer with an _action_, or with a _monitor_? How does it interact with monitors that have several actions (email / webhooks / ...)?
- If the code monitor action is "Send email to org", is the proposal that we maintain a blocklist of org members that have unsubscribed from a given monitor? Is that desirable?
- Is re-subscription possible? If so, how is it enabled?

**Comment by benvenker** [comment]
@lguychard I would agree, especially if we get to the expanded email recipients. There is still a good amount of consideration to be given to that feature and this definitely belongs in that list. I’ll make a note to add this to an epic.
